src/test/java/io/confluent/kafka/connect/datagen/DatagenConnectorTest.java

```diff
@@ -110,38 +111,66 @@ public void shouldAllowSettingSchemaFile() {
   @Test
   public void shouldFailValidationWithMultipleSchemaSources() {
     clearSchemaSources();
-    config.put(DatagenConnectorConfig.SCHEMA_STRING_CONF, "a schema");
-    config.put(DatagenConnectorConfig.SCHEMA_FILENAME_CONF, "a schema file");
-
+    config.put(DatagenConnectorConfig.SCHEMA_STRING_CONF,
+            "{\"namespace\":\"ksql\",\"name\":\"test_schema\",\"type\":\"record\",\"fields\":" +
+                    "[{\"name\":\"id\",\"type\":{\"type\":\"long\",\"arg.properties\"" +
+                    ":{\"iteration\":{\"start\":0}}}}]}");
```

Comment:
```suggestion
            "{\"namespace\":\"ksql\",\"name\":\"test_schema\",\"type\":\"record\",\"fields\":"
                    + "[{\"name\":\"id\",\"type\":{\"type\":\"long\",\"arg.properties\""
                    + ":{\"iteration\":{\"start\":0}}}}]}");
```
Sorry being nit-picky